flang/lib/Lower/Bridge.cpp

```diff
@@ -751,23 +751,27 @@ class FirConverter : public Fortran::lower::AbstractConverter {
         info.isStructured() ? builder->getIndexType() : info.loopVariableType;
     auto lowerValue = genFIRLoopIndex(info.lowerExpr, type);
     auto upperValue = genFIRLoopIndex(info.upperExpr, type);
-    info.stepValue = info.stepExpr.has_value()
-                         ? genFIRLoopIndex(*info.stepExpr, type)
-                         : info.isStructured()
-                               ? builder->create<mlir::ConstantIndexOp>(loc, 1)
-                               : builder->createIntegerConstant(
-                                     loc, info.loopVariableType, 1);
+    info.stepValue =
+        info.stepExpr.has_value() ? genFIRLoopIndex(*info.stepExpr, type)
+        : info.isStructured()
+            ? builder->create<mlir::ConstantIndexOp>(loc, 1)
+            : builder->createIntegerConstant(loc, info.loopVariableType, 1);
```

Comment:
This is annoying.  It looks like clang-format has changed fairly recently.

Reply:
> Does this just fix the zero-trip loop problem, or does it also fix the fm256 test 7 problem with an invalid lcv value after a loop with a positive iteration count?

It seemed to be fixed, but I will double check.

Reply:
> This is annoying. It looks like clang-format has changed fairly recently.

These arbitrary whitespace changes are just SOP when using a source formatting tool that changes. <sigh>